static/scss/general_styles.scss

```diff
@@ -0,0 +1,15 @@
+// General Style Rules
+
+body {
+  background-color: $page-background;
+  font-family: 'Roboto', helvetica, arial, sans-serif !important;
+  line-height: 1.4em;
+  }
+
+
+
+
+
+
+
+
```

Comment:
additionally, as George pointed out above css should generally be written like this:

```scss
selector {
  rule: value;
}
```

so that the closing brace is at the same indentation level as the first character in the selector, each rule is on it's own line, and any indentation is by 2 spaces at a time. This means a nested dealio would look like:

```scss
.foo {
  .bar {
    text-align: justify;
  }

  .baz {
    color: purple;
  }
}
```

I prefer to leave a blank line after a block (i.e. after the closing brace for the `.bar` block} unless I'm closing multiple blocks at once, in which case I think the braces should cascade (as they do after the `.baz` block).

let me know if that doesn't make sense, haha